README.md

```diff
@@ -116,57 +115,9 @@ Finally, properties are what give the selected elements of a rule declaration th
   - Uses of z-index
   - Compatibility or browser-specific hacks
 
-### OOCSS and BEM
+### Class Names
+TODO: How should we name classes?  My preference is something like `'section-element'` (ex. `'menu-block-title'`)
 
```

Comment:
I liked the BEM style of classes as it felt like less of a cognitive load and less nesting overall

Reply:
Yes probably less thinking when naming.  However, if you prepend your class with the section name, nesting isn't a problem.  However, I could be convinced on using BEM if that is the consensus.

Reply:
Can we use bem without mixing underscores and dashes?  Just use dashes. I don't like the mixed style, but don't mind the double dash style.

Reply:
If we just use dashes how do we use the "modifier" naming convention for different state or variation of sections?

Reply:
To be honest, I've never used bem or oocss, so not sure what you are asking.   I was just saying I like the style @ronsj  uses here: https://github.com/einstein/monolith/blob/e85d19b29f82b7bb44302eafd3414313cb01f174/templates/sections/features_list/features_list.php